lavalmi/tk-syntheyes
# PIP-843: Dedicated publishes in tk-multi-publish2 to export scene data to various file formats

Primary Changes:
- Added support for exporting camera, camera_tracks, object_tracks & geometry to .abc, .fbx, .ma, .nk & .usda.
-- **camera**: Only camera
-- **camera_track**: Only tracks corresponding to respective camera
-- **object_track**: Geometry and tracks of the respective object
-- **geometry**: Entire static geometry of the scene
-- NOTE: Depending on the exporter used, sometimes the exported items will be parented to another empty root node. Additionally, some files may contain data that was not filtered out entirely.
- When creating a new file via the tk-multi-workfiles2 app, the file selection-popup automatically defaults to the _Footage_ folder of the selected shot/asset/element. If the folder does not exist, the root directory of the project is selected.

Testing:
1. Start SynthEyes via ShotGrid
2. Open a saved file or create a new file and save it via the SynthEyes ShotGrid Panel.
3. Click on _Publish_ in the SynthEyes ShotGrid Panel
4. Select the desired exports in the publisher:
![grafik](https://github.com/user-attachments/assets/ada2fbb6-4700-46cd-947b-a5c91095f5ad)
5. Check the project folder to verify that SynthEyes exported the files correctly.

Required Repository:
- tk-config-lavapipeline
-- https://github.com/lavalmi/tk-config-lavapipeline/pull/135

Testing environment:
1. Configuration: matchmove-dev (see PR above)
5. SynthEyes Pro 2024 Build 1057 64-bit installed at "C:\Program Files\BorisFX\SynthEyes 2024\SynthEyes64.exe"
6. Default port range from 59200-59300
7. Shotgun python version: 3.7.11
